libs/ui-watt/src/lib/components/toast/index.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export { WattToastModule } from './watt-toast.module';
```

Comment:
question: why do we need the `WattToastModule` for? It seems like its only purpose is to provide the `MatSnackBarModule`. The other imports that are part of the module are needed for the `WattToastComponent`.

The `WattToastComponent`, on the other hand, is only relevant to `WattToastService` (i.e. internally in Watt).

So a wild (or maybe not) suggestion - let's convert the `WattToastComponent` to standalone. Convert `WattToastModule` to use the `forRoot` pattern and import it once in `DhCoreShellModule`. What do you think?

Reply:
I've now migrated the toast component to a standalone component. I think a `forRoot` module is a lot of boilerplate when we can just keep it as: 1. import the standalone component 2. inject the service.  
If the `forRoot` part is important to you I will ofcourse create that module :)